applications/rag/main.tf

```diff
@@ -218,6 +214,5 @@ module "frontend" {
   url_domain_addr          = var.frontend_url_domain_addr
   url_domain_name          = var.frontend_url_domain_name
   members_allowlist        = var.frontend_members_allowlist
-
-  depends_on = [module.cloudsql, module.gcs, module.inference-server]
+  depends_on = [ module.gcs ]
```

Comment:
We'll still need depend on module.cloudsql
rag-frontend deployment has cloud-sql-proxy which requires the CONNECTION_NAME.

Reply:
It should be fine then.